### ISSUE TITLE
Old Combat Mod causes crash (Minecraft 1.20.6, Fabric Loader 0.15.11)

### ISSUE THREAD

**Issue by MetroidCatcher** [TheHangel/old-combat-mod#1]
Directly after loading a world, the server immediately stops and Minecraft closes itself.

```
java.lang.NoSuchFieldError: Class net.minecraft.class_5134 does not have member field 'net.minecraft.class_1320 field_23723'
	at dev.hangel.old_combat_mod.OldCombatMain.changePlayersAttribute(OldCombatMain.java:36)
```

OS: Windows 10 IoT Enterprise LTSC 64bit, Version 21H2

The whole log is attached.

[crash-2024-05-20_11.55.17-server.txt](https://github.com/TheHangel/old-combat-mod/files/15375620/crash-2024-05-20_11.55.17-server.txt)

**Comment by TheHangel** [comment]
✅ Fixed on Minecraft 1.20.6 (Fabric)

[Commit fix](https://github.com/TheHangel/old-combat-mod/compare/1.20.x-fabric...1.20.6-fabric#diff-0ad8a418a494ef98969dc9ec6f7fc06758f28acabc2beac0d46383f5630db984)

Available on :

- [Modrinth](https://modrinth.com/mod/old-combat-mod/version/1aRfHzwQ)
- [Curseforge](https://www.curseforge.com/minecraft/mc-mods/old-combat-mod/files/5410528)

**Comment by TheHangel** [comment]
closed
